models/discordactions.js

```diff
@@ -47,16 +47,12 @@ const createNewRole = async (roleData) => {
 
 const removeMemberGroup = async (roleId, discordId) => {
   try {
-    const discordResponse = await removeRoleFromUser(roleId, discordId);
-    if (discordResponse) {
-      const backendResponse = await deleteRoleFromDatabase(roleId, discordId);
-      return backendResponse;
-    }
+    const backendResponse = await deleteRoleFromDatabase(roleId, discordId);
+    return backendResponse;
```

Comment:
- Why are we changing this? 
- Please correct me if I'm wrong but in these changes we are not checking response from discord to see if the role was actually removed from the user before removing it in our backend model.

Reply:
What is fx?

Reply:
Function